src/lib/chat-ui.js

```diff
@@ -501,6 +504,16 @@ const ChatUi = {
     });
   },
 
+  attachLinkEventListener() {
+    const links = document.querySelectorAll('.js-assistant-message a');
+    const link = links[links.length - 1];
```

Comment:
why do you select the last anchor tag, if there are a couple anchor tags?

Reply:
Bcs we should attach the handler to it, which sets the flag "chat seen" and close the chat. But should we define the chat as seen, if the link the user clicked is somewhere on the middle or at the beginnig of the chat, and we have other links below?

Reply:
Yes, I've discussed this with PMs. If we have more links comewhere in the chat (in last message or above) - we can allow to visitors just to click to them as it works for regular links, but the last link which is the same as in the button, should be tracked and the chat should be closed.